.github/workflows/rust.yml

```diff
@@ -143,3 +143,14 @@ jobs:
           pushd rust
           cargo doc --workspace --all-features
           popd
+      - name: Verify MSRV (Minimum Supported Rust Version)
+        if: matrix.os == 'ubuntu-latest'
+        working-directory: rust
+        run: |
+          curl -L "https://github.com/foresterre/cargo-msrv/releases/download/v0.18.4/cargo-msrv-x86_64-unknown-linux-gnu-v0.18.4.tgz" |
```

Comment:
(1) Could we perhaps validate the SHA so that upstream can't change it unexpectedly?
(2) Does it make sense to use `cargo install` instead (with a specific commit), or a specific version of their Docker image? (With Docker, we could even set this up so that it's run by docker-compose)

Reply:
Thanks for your comment. I too wondered if we needed to check hash.
(The protoc installed within this workflow does not seem to be checked, so I did not too)

How about 38e122351b943dfea67fbfb84ad55742c3aa372a?

- I considered a source install (similar to arrow-rs) first, but decided that a binary download would be better since it takes longer to build without a cache.
- I had not noticed this about Docker images, but it seemed difficult to choose which images to use because images are tagged and pushed at a higher frequency than releases.